core/genesis.go

```diff
@@ -324,8 +324,13 @@ func (g *Genesis) Commit(db ethdb.Database) (*types.Block, error) {
 	if err := config.CheckConfigForkOrder(); err != nil {
 		return nil, err
 	}
-	if config.Clique != nil && len(block.Extra()) == 0 {
-		return nil, errors.New("can't start clique chain without signers")
+	if config.Clique != nil {
+		if len(block.Extra()) < 32 {
+			return nil, errors.New("extra-data 32 byte vanity prefix missing")
+		}
```

Comment:
Seems to me that we can omit this one? If someone submits an empty extradata, it's a bit silly to say 
- "no sorry you need at least 32 bytes"
- "Ok here are 32 bytes"
- "no sorry you need at least 32+65 bytes" 

We might aswell error on the second clause from the beginning

Reply:
IMO the full change could have been to just change `len(block.Extra()) == 0` into `len(block.Extra()) < 32+crypto.SignatureLength` on line `327`.